src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/ResearchSetup.java

```diff
@@ -271,6 +271,7 @@ public static void setupResearches() {
         register("lead_clothing", 260, "Lead Clothing", 14, SlimefunItems.REINFORCED_CLOTH);
         register("tape_measure", 261, "Tape Measure", 7, SlimefunItems.TAPE_MEASURE);
         register("iron_golem_assembler", 262, "Automated Iron Golems", 30, SlimefunItems.IRON_GOLEM_ASSEMBLER);
+        register("unemployment_rune", 263, "Reset Villager Trades", 29, SlimefunItems.UNEMPLOYMENT_RUNE);
```

Comment:
Your Nether Goo should probably also be added to this research

Reply:
done